### PR TITLE
Resolve branch coverage in units.cpp

### LINKED ISSUE #1101: Clear missed branches from units.cpp

### PR DESCRIPTION
**Pull request by hsorby**
Fixes #1101.

### CODE REVIEW

File: tests/importer/importer.cpp [hunk 1]
```diff
@@ -1199,6 +1199,152 @@ TEST(Importer, importInvalidComponentFromCellmlModelResetError)
...
+    importedUnits1->setImportSource(importSource1);
+    importedUnits1->setImportReference("second_units");
+
+    importedUnits2->setImportSource(importSource2);
+    importedUnits2->setImportReference("third_units");
+
+    EXPECT_FALSE(model->hasUnresolvedImports());
+
+    auto flatModel = importer->flattenModel(model);
+    Debug() << importer->issue(0)->description();
```
Comment (agarny) [inline tests/importer/importer.cpp]: Delete.

**Review by agarny — CHANGES_REQUESTED** [review]
Looks good to me apart from a debug statement which should be removed.

File: tests/importer/importer.cpp (hunk 1 above)
Comment (hsorby) [reply tests/importer/importer.cpp]: Yeeks, who left that there!

Comment (hsorby) [reply tests/importer/importer.cpp]: [f436eff](https://github.com/cellml/libcellml/pull/1125/commits/f436eff9a16fec8ee9bae048d3dc1bb0b8633eea)

**Review by agarny — APPROVED** [review]

**Review by nickerso — APPROVED** [review]

**Pull request merged**